src/Files.App/Data/Items/WindowsCompatibilityOptions.cs

```diff
@@ -2,6 +2,7 @@
 // Licensed under the MIT License. See the LICENSE.
 
 using Files.Shared.Extensions;
+using RegexHelpers = Files.App.Data.Regex.RegexHelpers;
```

Comment:
I think you can put it into GlobalUsings class.

Reply:
Will be better, because `Regex` namespace conflict with the `Regex` class, causing all this weird usings. Tomorrow I will fix